### ISSUE TITLE
Test code field too short in custom metadata pilot

### ISSUE THREAD

**Issue by gemziebeth** [afawcett/declarative-lookup-rollup-summaries#428]
Hi 

I know it's not strictly good practice to have to cover the triggers with custom test code, but as another managed package is involved...

I have had to create rules in Classic DLRS due to the Test Code field only allowing 255 characters. Please could you revert this field back to being a Long text area so that we can make use of the custom metadata for our rule?

**Comment by afawcett** [comment]
This is a [duplicate of this issue](https://github.com/afawcett/declarative-lookup-rollup-summaries/issues/304). I'm waiting on a platform feature to support large text area fields, the 255 is a platform restriction at present. I could implement a workaround (a number of 255 text fields), but i want to hold out. My only suggestion is if your using another managed package (i'd love to know more details btw!) is to include a global method in the package that you can delegate to, thus the test code will fit into 255 characters.
